ic-os/components/networking/nftables/hostos/nftables.template

```diff
@@ -192,6 +200,8 @@ table ip6 filter {
     ip6 saddr @dfinity_dcs ct state { new } tcp dport { 22, 9100, 19531, 19100 } accept
     ip6 saddr @telemetry_clients ct state { new } tcp dport { 9100, 19531, 19100 } accept
     ip6 saddr @node_providers ct state { new } tcp dport { 22, 9100, 19531 } accept
+    ip6 saddr @local_prefix ct state { new } tcp dport { 9100, 19531, 19100 } accept
+    ip6 saddr @local_prefix udp dport { 9100, 19531, 19100 } accept
```

Comment:
Same question as above, but for HostOS.

Reply:
The list of ports looks good, and we don't need udp